lib/foods.js

```diff
@@ -44,26 +44,29 @@ function showRows(filter, rows) {
 }
 
 function addFoodOrWarning() {
-  $('#add-food').on('click', function() {
+  $('.add-food').on('click', function() {
     $('.validation-error').empty()
     var name = $('#name-field input').val();
     var calories = $('#calories-field input').val();
+    var table = $('#food-list')
```

Comment:
the convention is to name variables that are jQuery objects with a $ at the beginning to indicate that all jQuery methods are available on that variable 👍